pydantic_evals/pydantic_evals/_utils.py

```diff
@@ -101,3 +108,28 @@ async def _run_task(tsk: Callable[[], Awaitable[T]], index: int) -> None:
             tg.start_soon(_run_task, task, i)
 
     return results
+
+
+try:
+    from logfire._internal.config import (
+        LogfireNotConfiguredWarning,  # pyright: ignore[reportAssignmentType,reportPrivateImportUsage]
+    )
+# TODO: Remove this `pragma: no cover` once we test evals without pydantic-ai (which includes logfire)
```

Comment:
if logfire is a dependency now we should presumably just remove this whole try except thing, I know you didn't add this but we might as well remove it now

Reply:
nevermind I guess we didn't do that